tests/Exception/YotiClientExceptionTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoti\Test\Exception;
+
+use Yoti\Exception\YotiClientException;
+use Yoti\Test\TestCase;
+
+/**
+ * @coversDefaultClass \Yoti\Exception\YotiClientException
+ */
+class YotiClientExceptionTest extends TestCase
+{
+    public function testInstanceOfException()
+    {
+        $this->assertInstanceOf(\Exception::class, new YotiClientException());
```

Comment:
Test has been added for deprecated exception - this will be remove along with the exception in the next major